src/dmd/cpreprocess.d

```diff
@@ -0,0 +1,53 @@
+/**
+ * Run the C preprocessor on a C source file.
+ *
+ * Specification: C11
+ *
+ * Copyright:   Copyright (C) 2022 by The D Language Foundation, All Rights Reserved
+ * Authors:     $(LINK2 https://www.digitalmars.com, Walter Bright)
+ * License:     $(LINK2 https://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/cpreprocess.d, _cpreprocess.d)
+ * Documentation:  https://dlang.org/phobos/dmd_cpreprocess.html
+ * Coverage:    https://codecov.io/gh/dlang/dmd/src/master/src/dmd/cpreprocess.d
+ */
+
+module dmd.cpreprocess;
+
+import core.stdc.stdio;
+import core.stdc.string;
+
+import dmd.astenums;
+import dmd.globals;
+import dmd.link;
```

Comment:
Because, no dmd driver/backend code should be pulled into the front-end. (cc @kinke).

Reply:
offer a suggestion

Reply:
The Compiler interface might be one place for it. That is at least one module that is not meant to be common amongst implementing compilers.

Reply:
It's already been replaced with a pointer.

Reply:
What's imported from `dmd.link`?